BuberBreakfast/Model/Shop.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BuberBreakfast.Models;
+
+[Table("shop")]
```

Comment:
# Remark
Like a normal model, but with annotations to allow fields to map to column with diff name, and model to table with diff name.  
Else, dotnet would be expecting a column like `Name` instead of `name` in mysql, resulting in an error.

More info: https://www.entityframeworktutorial.net/code-first/column-dataannotations-attribute-in-code-first.aspx